packages/strapi-plugin-content-type-builder/admin/src/translations/es.json

```diff
@@ -24,29 +80,100 @@
   "form.attribute.item.number.type.decimal": "decimal (ex: 2.22)",
   "form.attribute.item.number.type.float": "flotante (ex: 3.3333333333)",
   "form.attribute.item.number.type.integer": "entero (ej: 10)",
+  "form.attribute.item.privateField": "Campo privado",
+  "form.attribute.item.privateField.description": "Este campo no aparecerpa en la respuesta de la API",
   "form.attribute.item.requiredField": "Campo obligatorio",
   "form.attribute.item.requiredField.description": "No podrá crear un registro si este campo está vacío",
   "form.attribute.item.settings.name": "Ajustes",
+  "form.attribute.item.text.regex": "Patrón RegExp",
+  "form.attribute.item.text.regex.description": "El texto de la expresión regular",
   "form.attribute.item.uniqueField": "Campo único",
   "form.attribute.item.uniqueField.description": "No podrá crear un registro si ya existe otro registro con el mismo contenido",
+  "form.attribute.media.allowed-types": "Seleccionar tipos de multimadia permitidos",
+  "form.attribute.media.allowed-types.none": "Ninguno",
+  "form.attribute.media.allowed-types.option-files": "Archivos",
+  "form.attribute.media.allowed-types.option-images": "imágenes",
+  "form.attribute.media.allowed-types.option-videos": "Videos",
+  "form.attribute.media.option.multiple": "Varios archivos Multimedia",
+  "form.attribute.media.option.multiple.description": "Mejor para sliders carruseles o descarga múltiple de archivos",
+  "form.attribute.media.option.single": "Un archivo multimedia",
+  "form.attribute.media.option.single.description": "Mejor para avatares, imagenes de perfil o portadas",
   "form.attribute.settings.default": "Valor por defecto",
+  "form.attribute.text.option.long-text": "Texto largo",
+  "form.attribute.text.option.long-text.description": "Mejor para descripciones, biografías. Las búsquedas exactas no están habilitadas",
+  "form.attribute.text.option.short-text": "Texto corto",
+  "form.attribute.text.option.short-text.description": "Mejor para títulos, nombres y vínculos (URL). La búsqueda exacta por palabra está disponible.",
+  "form.button.add.field.to.collectionType": "Agrega otro campo para este Contenedor",
+  "form.button.add.field.to.component": "Agrega otro campo para este Componente",
+  "form.button.add.field.to.contentType ": "Add another field to this content type",
+  "form.button.add.field.to.singleType": "Agrega otro campo para este Componente Único",
+  "form.button.add-components-to-dynamiczone": "Agrega un componente al área dinámica",
+  "form.button.add-field": "Agreg un nuevo campo",
+  "form.button.add-first-field-to-created-component": "Agrega el primer campo de tu componente",
```

Comment:
"Agrega un primero campo a este componente"

Reply:
It´s incorrect say "un primero campo"

Reply:
My bad, i change :)

Reply:
My bad, i change :)

Reply:
My bad, i change :)

Reply:
My bad, i change :)

Reply:
My bad, i change :)

Reply:
My bad, i change :)

Reply:
My bad, i change :)